src/Router.php

```diff
@@ -384,6 +384,9 @@ public static function process_http_request() {
 		 * to hook in to track metrics, such as how long the process took from `graphql_process_http_request`
 		 * to here, etc.
 		 *
+		 * @param $response array
+		 * @param $graphql_results array
```

Comment:
The type should actually come before the name of the variable for our inline doc parsing to pick up the type.